### PR TITLE
update TOS Link to be a parameter from brandingConfig

### LINKED ISSUE #4936: Update celo.org/terms in onboarding to differ based on the app
### Expected Behavior

celo.org/terms in onboarding should be replaced w/ the appropriate branding link

### Current Behavior

celo.org/terms is hardcoded

### PR DESCRIPTION
**Pull request by nityas**
### Description

This PR updates the terms link in onboarding to read from the brandingConfig

### Other changes

n/a

### Tested

not tested

### Related issues

- Fixes #4936 

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: packages/mobile/src/i18n.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {
   withTranslation as withTranslationI18Next,
 } from 'react-i18next'
 import * as RNLocalize from 'react-native-localize'
-import { APP_NAME } from 'src/config'
+import { APP_NAME, TOS_LINK } from 'src/config'
 import Logger from 'src/utils/Logger'
 
 const TAG = 'i18n'
```
Comment (annakaz) [inline packages/mobile/src/i18n.ts:14]:
```suggestion
const TAG = 'i18n'
const TOS_LINK_DISPLAY = TOS_LINK.replace(/^https?\:\/\//i, '')
```

File: packages/mobile/src/i18n.ts [hunk 2]
```diff
@@ -91,7 +91,7 @@ i18n
     debug: true,
     interpolation: {
       escapeValue: false,
-      defaultVariables: { appName: APP_NAME },
+      defaultVariables: { appName: APP_NAME, tosLink: TOS_LINK.replace(/^https?\:\/\//i, '') },
```
Comment (annakaz) [inline packages/mobile/src/i18n.ts]:
```suggestion
      defaultVariables: { appName: APP_NAME, tosLink: TOS_LINK_DISPLAY },
```

**Review by annakaz — APPROVED** [review]
LGTM! This change seem reasonable?

**Comment by Lss-Ankit** [comment]
Hi @jeanregisser i have verified this task using  Android internal build v1.1.0 (1004294309)& iOS test flight build v1.1.0(21)

- Verified Terms and conditions screen is open when user taps on Create new account / I have an account. 
- Verified  Terms of use and FAQ page is open on default mweb browser by tapping Valoraapp.com/terms link from Terms and conditions screen. 
- Verify user is redirect to Account Info screen by tapping on Accept button. 

Devices used: Google Pixel Xl (8.1) , iPhone 7+ (13.3.1) 

Please let me know if i nee to verify any thing else.

**Pull request merged**